ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/TracingOptionsManager.java

```diff
@@ -203,19 +191,17 @@ private Properties getOptions(IPluginModelBase model) {
 	 * support cases when: key is split in multiple lines; key use escape
 	 * characters; comment uses ! start char
 	 */
-	private void loadComments(InputStream stream, Properties modelOptions) throws IOException {
+	private static void loadComments(InputStream stream, Properties modelOptions) throws IOException {
 		String prevComment = ""; //$NON-NLS-1$
 		try (BufferedReader bufferedReader = new BufferedReader(
-				new InputStreamReader(stream, Charset.defaultCharset()))) {
-			String line;
-			while ((line = bufferedReader.readLine()) != null) {
+				new InputStreamReader(stream, StandardCharsets.ISO_8859_1))) {
```

Comment:
`Properties.store()` states in its java-doc that `The stream is written using the ISO 8859-1 character encoding.` and `Properties.load()` assume the same encoding. But if the InputStream is processed by a default Reader, that encoding has to be specified explicitly.

Reply:
Oh, I missed that, thanks for pointing it out! It makes sense then to specify the charset. I hope this doesn't bring problems in the future if someone edits the file with a text editor and (inadvertently) changes the charset.

I think some implementation detail like that would be worth being documented in a comment, something like `// the charset needs to be set explicitly because Properties.store() also uses this charset per default`

Reply:
> Oh, I missed that, thanks for pointing it out! It makes sense then to specify the charset. I hope this doesn't bring problems in the future if someone edits the file with a text editor and (inadvertently) changes the charset.

Then that's the fault of the person, but nothing we can do about. :/


> I think some implementation detail like that would be worth being documented in a comment, something like `// the charset needs to be set explicitly because Properties.store() also uses this charset per default`

Good point, done that.